### ISSUE TITLE
With verbosity 5 (or may be 4), report if # of loaded hashes per salt varies

### ISSUE THREAD

**Issue by frank-dittrich** [magnumripper/JohnTheRipper#1616]
If the distribution of hashes across salts varies, that information is interesting enough to be printed at least with higher verbosity.
May be it should also be in the log file.

(I'm talking about summaries here: "number of loaded hashes per salt varies from %d to %d\" ...)

Thoughts?

**Comment by magnumripper** [comment]
Agreed. At -verb=4 we could print the line you suggested. At -verb=5 we could even print distribution for each salt. Something similar to the below, manually made for Gawker dataset:
```
$ cut -d: -f2 gawker.txt | grep -E '^.{13}$' | cut -c1-2 | sort | uniq -c | sort -nr
 264 QS
 249 NC
 247 yV
 247 AA
 247 6s
 246 SO
 246 C8
 245 bL
 245 a4
 244 aE
(...)
```

**Comment by frank-dittrich** [comment]
For formats which (normally) should have just a single hash per salt, just a single dupe in a million hashes could create a very long list. So I am not sure I want to see that list with --verbosity=5.
When I get that warning (may be even at default verbosity), I can still use --show=left --salts= for more details.

**Comment by magnumripper** [comment]
In that scenario, what if we only list salts that has dupes?

**Comment by frank-dittrich** [comment]
I don't mind, but I also think it is not that useful. (Just use --show=left --salts= ... when you see the warning)
